### PR TITLE
Make 32bit incremental builds on linux not use free channel

### PR DESCRIPTION
**Pull request by stuartarchibald**
As title

### CODE REVIEW

**Comment by seibert** [comment]
looks good.

**Comment by seibert** [comment]
hmm, CI failing though.

File: buildscripts/incremental/setup_conda_environment.sh [hunk 1]
```diff
@@ -8,7 +8,11 @@ conda config --write-default
 conda config --set remote_connect_timeout_secs 30.15
 conda config --set remote_max_retries 10
 conda config --set remote_read_timeout_secs 120.2
-conda config --set restore_free_channel true
+if [[ $(uname) == Linux ]]; then
+    if [[ "$CONDA_SUBDIR" != "linux-32" || "$BITS32" != "yes" ]] ; then
```
Comment (seibert) [inline buildscripts/incremental/setup_conda_environment.sh]: I think this needs to be an `&&` rather than an `||`.

Comment (stuartarchibald) [reply buildscripts/incremental/setup_conda_environment.sh]: Thanks, I managed to invert only half the logic :)

**Comment by sklam** [comment]
Passed AzureCI

**Pull request merged**